File: dp_labs_and_solutions/dp_labs_and_solutions/dp_java_labs_and_solutions/clone/lab/OrganicLettuce.java
// OrganicLettuce.java

public class OrganicLettuce extends Lettuce
{
	public OrganicLettuce(double aPrice, int aNumberOfInsects)
	{
		super(aPrice);
		myNumberOfInsects = aNumberOfInsects;
	}

	public OrganicLettuce(double aPrice)
	{
		super(aPrice);
		myNumberOfInsects = 0;
	}

	public OrganicLettuce(OrganicLettuce aOrganicLettuce)
	{
		super(aOrganicLettuce);
		myNumberOfInsects = aOrganicLettuce.myNumberOfInsects;
	}

	private int myNumberOfInsects;
}

File: dp_labs_and_solutions/original/dp_labs_and_solutions/dp_cpp_labs_and_solutions/clone/lab/organic_lettuce.cpp
// organic_lettuce.cpp

#include "organic_lettuce.h"
#include "lettuce.h"

organic_lettuce::organic_lettuce(double a_price, int a_number_of_insects)
: lettuce(a_price)
{
	my_number_of_insects = a_number_of_insects;
}

organic_lettuce::organic_lettuce(const organic_lettuce & other)
: lettuce(other)
{
	my_number_of_insects = other.my_number_of_insects;
}

organic_lettuce::~organic_lettuce()
{
}
